langgraph_bot/nodes/anode/agentnode.py

```
@@ -9,17 +9,30 @@ def summary_agent_node(state:State):
+    last_message = f"""
+     and here  is the data :{state['data']}
+     here is the search result from internet: {state['tavily_search_result']}
+     """
+
+    response = agent.invoke({"messages":[HumanMessage(content=last_message)]})
+
+    try:
+        if response:
+            description = response['messages'][-1].content
+            return {"messages":response["messages"],"description":description,"description_agent_success":True}
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)